frontend/src/app/chart-item/chart-item.component.scss

```diff
@@ -31,17 +32,26 @@ $chart-item-logo-width: 80px;
     // Remove this when chips are ready:
     // https://github.com/angular/material2/issues/120
     &__repo {
-      color: mat-color($monocular-app-accent);
-      font-size: .8em;
-      padding: 2px 4px;
+      color: lighten($layout-base, 10%);
+      padding: 2px 1px 2px 0px;
       border-radius: 4px;
 
       &:hover {
         background: $background-white;
       }
 
-      &.repo-incubator {
-        color: mat-color($monocular-app-warn, 600);
+      &__namespace {
+        margin-right: -4px;
+        color: mat-color($monocular-app-primary, 600);
+
+        &.repo-incubator {
+          color: mat-color($monocular-app-warn, 600);
+        }
+
+        &::after {
+          margin-left: -4px;
+          content: "/";
```

Comment:
Any reason to use `:after` instead of just placing the `/` in the HTML? Just curious, I'm not sure what the browser support is for `:after`, though it's probably well supported at this point.

Reply:
Ran into issues earlier but seems to work just fine now. 😄

Reply:
I could go back to the much simpler: https://github.com/helm/monocular/pull/599/commits/a420c8740951ff39948b8cbb1213ee97d97e5a05

If we removed: https://github.com/helm/monocular/pull/599/files#diff-5ef69502030bb0c201b064c092ccd0caR45

What do you think? Is this `:hover` necessary?